dist/tools/openocd/openocd.sh

```diff
@@ -191,8 +191,16 @@ do_debug() {
     test_elffile
     test_ports
     test_tui
+    # setsid is needed so that Ctrl+C in GDB doesn't kill OpenOCD
+    [ -n "$(which setsid)" ] && SETSID=setsid
```

Comment:
```bash
[ -n "${SETSID}" ] && SETSID=$(which setsid)
```
I don't get this. If `$SETSID` is not empty, then set it to the output of `which setsid`? Why should `$SETSID` be set in the first place?